tldw_Server_API/app/core/Sharing/unified_share_audit.py

```
@@ -0,0 +1,315 @@
+                f"""
+                CREATE TABLE IF NOT EXISTS {_SHARE_AUDIT_STATE_TABLE} (
+                    key TEXT PRIMARY KEY,
+                    int_value INTEGER NOT NULL
+                )
+                """
+            )
+            await db.execute(
+                f"""
+                INSERT OR IGNORE INTO {_SHARE_AUDIT_STATE_TABLE} (key, int_value)
+                VALUES (?, 0)
+                """,
```

Ruff (0.15.9)

[error] Possible SQL injection vector through string-based query construction

(S608)